tests/test_date_tools.py

```diff
@@ -5,34 +5,38 @@
 
 from portal.date_tools import FHIR_datetime, RelativeDelta, localize_datetime
 from tests import TestCase
+import pytest
```

Comment:
Can you put this 3rd party import in its own block, above the `portal` import line?